### ISSUE TITLE
For ovms workload getModels.sh not working when it is called by docker-run.sh from project base directory

### ISSUE THREAD

**Issue by jim-wang-intel** [intel-retail/automated-self-checkout#96]
## Version

What release version was the issue found in? 1.5.0

## Hardware configuration
12th Gen Intel® Core™ i7

List Hardware components used. 
Ex. 
Issue found with the following:
- Intel® xeon Platinum 8351N with Intel® Data Center GPU Flex 140
- 12th Gen Intel® Core™ i7 with Intel® Arc™ A770M Graphics

## Describe the bug

under the base directory, when run docker-run.sh with ovms workload, the `./getModels.sh` script gives the error when it is trying to call `./downloadOVMSModels.sh`

## To Reproduce

1. Go to automated-self-checkout project base directory
2. Run the camera-simulator if not running yet:
```bash
sudo ./camera-simulator/camera-simulator.sh
```
3. Run with the docker-run.sh with the following:
```bash
$ sudo ./docker-run.sh --workload opencv-ovms --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0 --ocr_disabled --barcode_disabled --classification_disabled
```
and it gave the following error message on the console:
```console
Starting open-ovms model download...
./download_models/getModels.sh: line 63: ./downloadOVMSModels.sh: No such file or directory
```

**Comment by jim-wang-intel** [comment]
Note: this also failed for dlstreamer model downloading as well ... as long as the ./docker-run.sh is run from the project root directory, both download scripts failed as its runtime execution path is implemented wrong

**Comment by jim-wang-intel** [comment]
Closed by PR #98
